ResearchKit/Common/ORKHelpers.h

```diff
@@ -34,6 +35,37 @@
 #import <ResearchKit/ORKErrors.h>
 
 
+#if ( defined(ORK_LOG_LEVEL_NONE) && ORK_LOG_LEVEL_NONE )
+#  undef ORK_LOG_LEVEL_DEBUG
+#  undef ORK_LOG_LEVEL_WARNING
+#  undef ORK_LOG_LEVEL_ERROR
+#endif
+
+#if ( !defined(ORK_LOG_LEVEL_NONE) && !defined(ORK_LOG_LEVEL_DEBUG) && !defined(ORK_LOG_LEVEL_WARNING) && !defined(ORK_LOG_LEVEL_ERROR) )
+#  define ORK_LOG_LEVEL_WARNING 1
```

Comment:
That is correct, if none of the *log levels* is specified, it defaults to `ORK_LOG_LEVEL_WARNING`. I think this is saner than defaulting to `ORK_LOG_LEVEL_LOG` or to `ORK_LOG_LEVEL_NONE`.

I have detailed this on the documentation.

What do you think?